src/UI/Component/Item/Item.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Item;
+
+/**
+ * Common interface to all items.
+ */
+interface Item extends \ILIAS\UI\Component\Component {
+	/**
+	 * Gets the title of the item 
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Create a new item with an attached description.
+	 * @param string $description
+	 * @return AppointmentItem
+	 */
+	public function withDescription($description);
+
+	/**
+	 * Get the description of the item.
+	 * @return string
+	 */
+	public function getDescription();
+
+	/**
+	 * Get a new item with the given properties as key-value pairs.
+	 *
+	 * The key is holding the title and the value is holding the content of the
+     * specific data set.
+	 *
+	 * @param array<string,string> $properties Title => Content
+	 * @return self
+	 */
+	public function withProperties(array $properties);
+
+	/**
+	 * Get the properties of the appointment.
+	 *
+	 * @return array<string,string>		Title => Content
+	 */
+	public function getProperties();
+
+	/**
+	 * Create a new appointment item with a set of actions to perform on it.
+	 *
+	 * @param string[] $actions
+	 * @return AppointmentItem
+	 */
+	public function withActions(array $actions);
+
+	/**
+	 * Get the actions of the item.
+	 *
+	 * @return string[]
+	 */
+	public function getActions();
+
+	/**
+	 * Set a marker id. Marker IDs must be >=0 and <=32. They assign the item to a CSS class il-item-marker-[ID].
+	 * Set 0 for no marker (default).
+	 *
+	 * @param int $marker_id
+	 */
+	public function withMarkerId($marker_id);
+
+	/**
+	 * @return int
+	 */
+	public function getMarkerId();
+
+	/**
+	 * Set image as lead
+	 *
+	 * @param \ILIAS\UI\Component\Image\Image $image lead image
+	 */
+	public function withLeadImage(\ILIAS\UI\Component\Image\Image $image);
```

Comment:
You are indeed right with the blog posting. But we do not have these things for a forum posting, as we have more options on the left side there, right?

@Amstutz and I discussed this question about the knowledge the UI framework should have as well and I think we agree in general: the UI framework should not have knowledge on things in ILIAS. But we also lose the possibility to address (e.g.) "blog postings" only without addressing all other items as well if we do not introduce these (or take care about the problem by other means). Also a "blog posting" might in fact have a distinguishable visualisation which is worth a new name. One might also argue, that "blog" and "forum" are indeed not names from ILIAS but general concepts (akin to "appointment"), which would circumvent the problem on another level. So there is room for interesting debates here.